content/efcore/getting-started/querying.md

```diff
@@ -65,7 +65,7 @@ Loading all of the entities from a database is useful, but there are many use ca
 using (var context = new LibraryContext()) 
 { 
     var books = context.Books 
-        .Where(b => b.Genre == "Historical") 
+        .Where(b => b.Genre == "Historical")
```

Comment:
Yeah, it looks like I messed up and accidentally deleted the wrong thing when resolving the conflicts. Give me a sec.